### PR TITLE
refactor: Remove title bar for linux

### PR DESCRIPTION
**Pull request by Xuanwo**
image

### CODE REVIEW

**Comment by yetone** [comment]
LGTM!

**Comment by yetone** [comment]
Maybe we also need to reduce the padding-top: https://github.com/yetone/openai-translator/pull/154/files#diff-fb07df317723a760a3e04a60c492fdd0fe6dc8028ad438eb000eca45d8a1a898L69

**Comment by Xuanwo** [comment]
> Maybe we also need to reduce the padding-top: https://github.com/yetone/openai-translator/pull/154/files#diff-fb07df317723a760a3e04a60c492fdd0fe6dc8028ad438eb000eca45d8a1a898L69

Are you suggesting that we simply remove it? I apologize, but I am not familiar with frontend development. Could you please provide clear guidance?

**Comment by Xuanwo** [comment]
image

Seems cool!

**Comment by yetone** [comment]
> > Maybe we also need to reduce the padding-top: https://github.com/yetone/openai-translator/pull/154/files#diff-fb07df317723a760a3e04a60c492fdd0fe6dc8028ad438eb000eca45d8a1a898L69
> 
> Are you suggesting that we simply remove it? I apologize, but I am not familiar with frontend development. Could you please provide clear guidance?

Yes, we can remove it if is Linux

**Comment by yetone** [comment]
> image
> 
> Seems cool!

Cool!

File: src/tauri/App.tsx [hunk 1]
```diff
@@ -65,9 +66,6 @@ export function App() {
                 autoFocus
                 defaultShowSettings
                 editorRows={10}
-                containerStyle={{
```
Comment (yetone) [inline src/tauri/App.tsx:68]: Should check if it is a Linux system.

Comment (Xuanwo) [reply src/tauri/App.tsx:68]: Fixed, PTAL.

**Pull request merged**